lib/iris/aux_factory.py

```diff
@@ -296,7 +296,6 @@ def _remap_with_bounds(self, dependency_dims, derived_dims):
                     # extra dimension to make the shape compatible, so
                     # we just add an extra 1.
                     shape.append(1)
-                nd_values = np.array(nd_values)
```

Comment:
We can use test coverage to convince ourselves that his is a sensible change ...